testing/py-protobuf/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Maintainer: Corentin Henry <corentinhenry@gmail.com>
+pkgname=py-protobuf
+_pkgname=protobuf
+pkgver=2.6.1
+pkgrel=0
+pkgdesc="Protocol Buffers are Google’s data interchange format."
+url="https://github.com/google/protobuf"
+arch="noarch"
+license="Copyright 2014, Google Inc. All rights reserved."
+depends="python"
```

Comment:
Module protobuf [has some Python requirements](https://github.com/google/protobuf/blob/master/python/setup.py#L213) that must be also specified in `depends`. It seems that only `six` and `setuptools` are needed, `ordereddict` is not needed for Python 2.7 and `unittest2` is needed only for tests.